<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customTripPage</class>
 <widget class="QWidget" name="customTripPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Cities</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="cityList"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Planned Cities</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="tripList"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="finishBtn">
     <property name="styleSheet">
      <string notr="true"> QPushButton{
        margin-left: 50px; margin-right: 50px;
        background-color: rgb(146, 189, 108);
         border-style: solid;
         border-width: 2px;
         border-color: beige;
         font: bold 14px;
		  color: rgb(0, 0, 0);
     }
  QPushButton:pressed {
         background-color: rgb(224, 0, 0);     
     }
     QPushButton:hover {
         background-color: rgb(224, 255, 0);
     }

    QPushButton:hover:pressed
    {
        background-color:red;
    }</string>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="addBtn">
     <property name="styleSheet">
      <string notr="true"> QPushButton{
        margin-left: 50px; margin-right: 50px;
        background-color: rgb(146, 189, 108);
         border-style: solid;
         border-width: 2px;
         border-color: beige;
         font: bold 14px;
		  color: rgb(0, 0, 0);
     }
  QPushButton:pressed {
         background-color: rgb(224, 0, 0);     
     }
     QPushButton:hover {
         background-color: rgb(224, 255, 0);
     }

    QPushButton:hover:pressed
    {
        background-color:red;
    }</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="removeBtn">
     <property name="styleSheet">
      <string notr="true"> QPushButton{
        margin-left: 50px; margin-right: 50px;
        background-color: rgb(146, 189, 108);
         border-style: solid;
         border-width: 2px;
         border-color: beige;
         font: bold 14px;
		  color: rgb(0, 0, 0);
     }
  QPushButton:pressed {
         background-color: rgb(224, 0, 0);     
     }
     QPushButton:hover {
         background-color: rgb(224, 255, 0);
     }

    QPushButton:hover:pressed
    {
        background-color:red;
    }</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="backBtn">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;go to previous the page&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
